.github/workflows/CI-tag-orchestrator.yml

```diff
@@ -73,35 +151,35 @@ jobs:
             echo "JOB_STATUS=success" >> $GITHUB_ENV
           fi
 
-      - name: Notify Slack
-        if: ${{ env.JOB_STATUS == 'failed' && !env.ACT }}
-        uses: slackapi/slack-github-action@v1.25.0
-        with:
-          payload: |
-            {
-              "blocks": [
-                {
-                  "type": "header",
-                  "text": {
-                    "type": "plain_text",
-                    "text": "🚨 ALERT - ZkVerify CI/CD Tag Orchestrator Job Failed! 🚨"
-                  }
-                },
-                {
-                  "type": "section",
-                  "fields": [
-                    {
-                      "type": "mrkdwn",
-                      "text": "*Job URL:* <https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}|View Job>"
-                    },
-                    {
-                      "type": "mrkdwn",
-                      "text": "*Pull Request:* <${{ github.event.pull_request.html_url || github.event.head_commit.url }}|View PR/Commit>"
-                    }
-                  ]
-                }
-              ]
-            }
-        env:
-          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+  #     - name: Notify Slack
```

Comment:
Still WIP for the commented out lines?

Reply:
I prefer if you remove them instead of comment out: git stand here exactly for doing this job :smile: